__tests__/endpoints/metrics.test.js

```diff
@@ -89,3 +92,26 @@ describe("GET /metrics/resourceCount", () => {
     expect(mockResponse).toHaveBeenCalledWith(resourceOnlyQuery)
   })
 })
+
+describe("GET /metrics/createdCount", () => {
+  it("adds correct filters for date and resource type", async () => {
+    const mockCollection = (collectionName) => {
+      return {
+        resources: { aggregate: jest.fn().mockResolvedValue([response]) },
+      }[collectionName]
+    }
+    const mockDb = { collection: mockCollection }
+    connect.mockImplementation(mockConnect(mockDb))
+
+    const res = await request(app)
+      .get("/metrics/createdCount/all?startDate=2021-10-01&endDate=2021-11-01")
+      .set("Accept", "application/json")
+    console.log(res)
+    expect(res.statusCode).toEqual(200)
+    expect(res.type).toEqual("application/json")
+    expect(res.body).toEqual(response)
+    expect(mockResponse).toHaveBeenCalledWith(
+      expect.objectContaining({ $match: allResourceQuery })
+    )
```

Comment:
Need help here, I don't want to add an expectation for the entire big `aggregation` query, was hoping to just verify some subset of it, but can't get the syntax.